packages/core/src/amazonqGumby/chat/controller/messenger/messenger.ts

```diff
@@ -328,6 +337,28 @@ export class Messenger {
         )
     }
 
+    public sendUnrecoverableError(type: UnrecoverableErrorResponseType, tabID: string) {
+        let message = '...'
+
+        switch (type) {
+            case 'no-alternate-dependencies-found':
+                message = `I couldn't find any other versions of this dependency in your local Maven repository. Try transforming the depedency to make it compatible with Java 17, and then try transforming this module again.`
+                break
+        }
+
+        this.dispatcher.sendChatMessage(
+            new ChatMessage(
+                {
+                    message,
+                    messageType: 'ai-prompt',
+                },
+                tabID
+            )
+        )
+
+        this.sendInProgressMessage(tabID, 'Stopping job...')
```

Comment:
This isn't needed anymore, but I'll remove this in a separate PR / when we're able to test UX again